de-lib/src/main/java/org/iplantc/de/client/models/collaborators/OldCollaborator.java

```diff
@@ -9,9 +9,9 @@
 
 /**
  * @author sriram
- * 
+ *
  */
-public interface Collaborator extends HasId {
+public interface OldCollaborator extends HasId {
```

Comment:
Yes, we still have a few endpoints that the UI needs that is still returning the old `Collaborator` format which has different keys than a `Subject`.  For example, iplant-groups doesn't currently have an endpoint which will return a user's list of plain old Collaborators that do NOT belong to a list.  So we're still using the old apps endpoint which has the old format.